clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -9364,6 +9364,14 @@ def warn_atomic_implicit_seq_cst : Warning<
   InGroup<DiagGroup<"atomic-implicit-seq-cst">>, DefaultIgnore;
 def err_atomic_unsupported : Error<
   "atomic types are not supported in '%0'">;
+def warn_cv_stripped_in_enum : Warning<
+  "%select{'const' and 'volatile' qualifiers|'const' qualifier|'volatile' "
```

Comment:
Seems like a perfect time for an `enum_select` :)

Reply:
Good call!